src/app/shell/personal-cabinet/shared-cabinet/messages/chat/message/message.component.html

```diff
@@ -2,7 +2,8 @@
   <h4 class="sender" *ngIf="isFirstMessage">
     {{ senderName }}
   </h4>
-  <div [ngClass]="message.senderRoleIsProvider === userIsProvider ? 'message message__outcome' : 'message message__income'">
+  <div
+    [ngClass]="message.senderRoleIsProvider === userIsProvider ? 'message message__outcome' : 'message message__income'">
```

Comment:
class `message` , which is used in both cases, can be moved to the `class` attribute